### PR TITLE
Add dictionary definition for "Ubantu"

### PR DESCRIPTION
**Pull request by nik132-eng**
## This PR fixes...

This PR fixes the Dictionary definition of  "Ubantu"

## What I did...

- Add dictionary definition for "Ubantu".
- Add brain's perspective.

## How to test...

_In a list format, tell us how to test this PR._
Ex:

1. Navigate to /Dictionary
2. Search "Ubantu" 
3. See the definition and perspective show up under "Ubantu"
4. Check grammar and typos

## I learned...

learnt perfect definition

### CODE REVIEW

**Review by alliequintano — CHANGES_REQUESTED** [review]
Hi @nik132-eng, thank you for contributing to our tech dictionary! I like your idea for the perspective from the role of the operating system; however, we are looking for roles more like roles on a software team - like developer, designer, QA, product manager, etc. 

Are you working as or interested in one particular role? I suggest you use that are your role and write a new meaning from that perspective. Thank you!

File: src/assets/content/dictionary/ubuntu.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ---
-title: Ubuntu
-definition: ''
-perspectives: []
-
+title: Ubantu
```
Comment (alliequintano) [inline src/assets/content/dictionary/ubuntu.md]:
```suggestion
title: Ubuntu
```

**Review by alliequintano — CHANGES_REQUESTED** [review]
You can accept my suggestion to correct this small spelling mistake.

**Comment by nik132-eng** [comment]
can you mention some issues @alliequintano ?

**Comment by alliequintano** [comment]
Hi @nik132-eng, please see my previous comment about updating your 'perspective'
>Hi @nik132-eng, thank you for contributing to our tech dictionary! I like your idea for the perspective from the role of the operating system; however, we are looking for roles more like roles on a software team - like developer, designer, QA, product manager, etc.
>Are you working as or interested in one particular role? I suggest you use that are your role and write a new meaning from that perspective. Thank you!

The example in the [CONTRIBUTING.md](https://github.com/cherryontech/website/blob/pit/CONTRIBUTING.md) has a helpful example of the formatting that's needed and how to think about it. I also suggest following the directions listed there for testing locally or via the Netlify deploy preview in order to read your sentence:
image
This helps you catch mistakes like the additional period and capitalization, as well as reading the full sentence for clarity.

**Pull request closed without merging**